mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -264,6 +264,47 @@ func TestAccContainerCluster_withConfidentialNodes(t *testing.T) {
 }
 <% end -%>
 
+<% unless version == 'ga' -%>
+func TestAccContainerCluster_withILBSubSetting(t *testing.T) {
```

Comment:
super nit: Feel free to use use "ILBSubsetting" is probably the casing I'd use here